### PR TITLE
Feature/infinite moving cards

### PR DESCRIPTION
**Pull request by sivareddyuppathi**

### CODE REVIEW

File: content/2.components/infinite-moving-cards.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: Infinite Moving Cards
+description: A customizable group of cards that move infinitely in a loop. Made with Framer Motion and Tailwind CSS.
```
Comment (rahul-vashishtha) [inline content/2.components/infinite-moving-cards.md:3]: We are not using Framer Motion.

File: eslint.config.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export default withNuxt(
       "import/newline-after-import": "error",
       "import/no-duplicates": "error",
       "import/no-unresolved": "off", //Need eslint-import-resolver-typescript, waiting for flatconfig and error fix on package side
-      "func-style": ["error", "declaration"],
+      "func-style": ["error", "declaration", { allowArrowFunctions: true }],
```
Comment (rahul-vashishtha) [inline eslint.config.js:53]: Arrow functions are not allowed.

**Review by rahul-vashishtha — COMMENTED** [review]
Hi @sivareddyuppathi,
Thanks for providing this component.

After reviewing, I found that this component seems like another implementation of Marquee component we already have. It has the same functionality and features.

https://inspira-ui.com/components/marquee

I think there is no need to have 2 components which are exactly similar in features & functionality.

**Comment by sivareddyuppathi** [comment]
Yeah you're right @rahul-vashishtha I'm closing this PR.

**Pull request closed without merging**